framework/include/base/XFEMInterface.h

```diff
@@ -48,9 +49,9 @@ class XFEMInterface : public ConsoleStreamInterface
    * Constructor
    */
   explicit
-  XFEMInterface(MooseApp & app, const MooseSharedPointer<FEProblem> fe_problem):
-    ConsoleStreamInterface(app),
-    _fe_problem(fe_problem),
+  XFEMInterface(const InputParameters & params) :
+    ConsoleStreamInterface(*params.get<MooseApp *>("_moose_app")),
```

Comment:
Use this method instead:
https://github.com/idaholab/moose/blob/devel/framework/include/utils/InputParameters.h#L112

Same for the line of code below